test/Microsoft.AspNetCore.Server.Kestrel.FunctionalTests/project.json

```diff
@@ -31,6 +31,11 @@
   },
   "buildOptions": {
     "allowUnsafe": true,
+    "compile": {
+      "include": [
+        "../shared/**/*.cs"
```

Comment:
Surprised you didn't put this in a comment test project. Not that I'm complaining :smile: